doc/userguide/devguide/codebase/contributing/code-submission-process.rst

```diff
@@ -19,6 +19,7 @@ Commits
     * Description, wrapped at ~72 characters
 #. Commits should be individually compilable, starting with the oldest commit. Make sure that
    each commit can be built if it and the preceding commits in the PR are used.
+#. Commits should be signed with the format: "FirstName LastName <name@example.com>"
```

Comment:
Authored or signed by?  I know I sometimes see commits with the signed off line at the bottom, but in this case we're specifically referring to the the authored by line I think.

Reply:
I see. I thought they were synonymous and "Signed off by" thingy was used by patch submitters in a non GUI setting like a mailing list. Will fix. Thanks!